developers/weaviate/model-providers/_includes/provider.connect.py

```diff
@@ -79,6 +87,13 @@
 # START CohereInstantiation
     "X-Cohere-Api-Key": cohere_key,
 # END CohereInstantiation
+# START FriendliInstantiation
+    "X-Friendli-Api-Key": friendli_key,
+# END FriendliInstantiation
+# START FriendliDedicatedInstantiation
+    "X-Friendli-Api-Key": friendli_key,
+    "X-Friendli-Baseurl": "https://inference.friendli.ai/dedicated",
```

Comment:
default base url defined in Weaviate module is `https://inference.friendli.ai` here is different, is this OK?

Reply:
This is OK. 
This is right configuration for users who want to use their own models deployed on Friendli Suite :)